src/modules/esl-footnotes/core/esl-note.ts

```diff
@@ -46,9 +46,17 @@ export class ESLNote extends ESLBaseElement {
     return ['tooltip-shown'];
   }
 
-  get index(): number {
+  public get index(): number {
     return this._index;
   }
+  public set index(value: number) {
+    this._index = value;
+    this.innerHTML = this.renderedIndex;
+  }
+
+  public get renderedIndex(): string {
+    return `${this._index}`;
+  }
```

Comment:
Is it in purpose to be rewritten?

Reply:
That's right.

Reply:
And since the rewritten value must be used by the footnote component.